sass/directives/03_components/cards/_cards.scss

```diff
@@ -115,3 +115,142 @@
     @include flex-wrap(nowrap);
   }
 }
+
+// Galleries with filter
```

Comment:
Let's do away with the comment and move the gallery-related card mixins to a new file, instead. Thoughts?

Reply:
Honestly, I would be up for removing the gallery namespace altogether, use a more generic namespace, and keep the styles in this file. These mixins are exactly related to the gallery component or a gallery in general. They're just modified styles of the mixins above to show a new style of card.

Reply:
I'm fine with that - what do you think would be a good name for these mixins?

Reply:
Here's what I had in mind:
- For `card--tile__gallery-item` -> `card--tile--basic` because this is essentially a card that scales but does not include any display features like flexbox to center content, does not include the strip, does not include the box shadow, and only contains styles to correctly position itself within its parent flex container `card-deck--centered`
- For `card--tile__gallery-image` -> `card--tile__image($include_box-shadow: true)` since this is just adding styles to correctly size the image within the card tile. The option enables/disables a box shadow around the image
- For `card--tile__gallery-subtext` -> `card--tile__caption` since the item currently using this is just a simple header with descriptive text below it. Its not a header however. It really is just a simple caption.
-For `card-deck--controls ` -> `card--deck__filters` since this mixin styles the filters seen on the gallery widget. Could be used with any `card--deck` that is controlled by filters.
- For `card--deck__popup` -> `sample-product--modal`. We can move this to a different file. As the name says, this would be to style a modal to display an image of a product with a link at the bottom.

Reply:
These refactors sound excellent and well-thought-out. Let's do it!